app/models/financialDetails/BalanceDetails.scala

```diff
@@ -42,6 +43,10 @@ case class BalanceDetails(balanceDueWithin30Days: BigDecimal,
     availableCredit.map (credit => math.abs(credit.toDouble))
   }
 
+  def getAbsoluteAllocatedCreditAmount: Option[BigDecimal] = {
+    allocatedCredit.map(credit => math.abs(credit.toDouble))
```

Comment:
we need to raise this with the BAs to make sure we are doing correct calculation